### DISCUSSION TITLE
Alternative for spaces: Window Tags?

### DISCUSSION THREAD

**Discussion by adwinying** [asmvik/yabai#1414] [Ideas]
## Background

Yabai is working great for me, but the space switching animation is really bothering me. 
Seems like there's no way to disable the animation without disabling SIP, which isn't an option for me (especially on a work device).

## Idea

And then it came to me: instead of managing windows using macOS's Desktops, how about managing windows with tags instead (like AwesomeWM or dwm)? 

## Concept

For a single space, windows can be tagged with different number, and activating the tag number would show windows only with the said tag

### Pros
- A workaround for disabling SIP

### Cons
- No integration with macOS desktops (Mission control, etc.)

## Questions

- Would something like thing be a feature that yabai could implement?
- If not, would there be any other ways to implement something like this?

**Comment by asmvik** [comment]
It is possible to implement this on macOS yes, but it is not something that I am going to do in yabai, sorry.
